src/goto-programs/class_hierarchy.cpp

```diff
@@ -123,17 +126,17 @@ void class_hierarchyt::get_parents_trans_rec(
     get_parents_trans_rec(child, dest);
 }
 
-void class_hierarchyt::output(std::ostream &out) const
+void class_hierarchyt::output(std::ostream &out, bool children_only) const
```

Comment:
Please document - it isn't obvious to me what `children_only` means without reading the code.